nodeup/pkg/model/kube_proxy.go

```diff
@@ -204,6 +204,7 @@ func (b *KubeProxyBuilder) buildPod() (*v1.Pod, error) {
 	{
 		addHostPathMapping(pod, container, "kubeconfig", "/var/lib/kube-proxy/kubeconfig")
 		addHostPathMapping(pod, container, "logfile", "/var/log/kube-proxy.log").ReadOnly = false
+		addHostPathMapping(pod, container, "modules", "/lib/modules")
```

Comment:
@gambol99 I think we should add a comment as well.  The reason for this mapping is less obvious than the other two.

Reply:
yep, make's sense .. I've added a note